lib/js/src/rpc/messages/AddCommand.js

```diff
@@ -74,7 +74,7 @@ class AddCommand extends RpcRequest {
     }
```

Comment:
the param for  `setMenuParams` is object, so shouldn't we use `this.setValue(AddCommand.KEY_MENU_PARAMS, menuParams);` instead of `setParameter `?

Reply:
There is no method setValue.